Adds custom tags.yml and clothing

Colin-Tel opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

Adds a tags.yml to Prototypes/DeltaV as well as adding the custom DeltaV and Nyano clothing protos and textures.

**From this point on, we should put any non-upstream tags in `Resources/Prototypes/DeltaV/tags.yml`.**

Any instance of currently-missing components was commented out. I should have gotten them all, as my local tests have passed.

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

:cl:
- add: Added Delta-V and Nyano clothing.

DebugOk commented:
This was merged into Nyano with https://github.com/Nyanotrasen/Nyanotrasen/pull/1541
Daemon will probably pull that cherry pick here too, so this pull would be included

Colin-Tel committed:
Adds custom tags.yml and clothing

Adds a tags.yml to Prototypes/DeltaV as well as adding the custom DeltaV and Nyano clothing protos and textures.
